code/api/views.py

```diff
@@ -109,9 +110,51 @@ def get(self, request, author_id, post_id):
 
 @method_decorator(csrf_exempt, name='dispatch')
 class PostCommentsView(View):
+    '''
+        HANDLE Comment GET and POST
+    '''
 
     def get(self, request, author_id, post_id):
-        return HttpResponse("This is the authors/aid/posts/pid/comments/ endpoint")
+        # Send all comments 
+        # TODO: Update json response and data serialization to match spec
+        try:
+            comments = Comment.objects.filter(post=post_id).order_by('-pub_date')
+            
+            serializedComments = []
+            for comment in comments:
+                serialized_obj = serializers.serialize('json', [ comment, ])
+                serializedComments.append(serialized_obj)
+        except Exception as e:
+            print(e)
+            return HttpResponseServerError()
+          
+        return JsonResponse(serializedComments, safe=False)
+
+    def post(self, request, author_id, post_id):
+        # check if authenticated
+        if (not request.user):
+            return HttpResponseForbidden()
+
+        comment = request.POST.get('comment')
```

Comment:
Can you take a look at the way I parsed JSON input in the inbox/ endpoint and compare? I'm not saying anything is wrong with this, I'm just not sure if the API is supposed to receive data using request.POST.get(param), or the data should be stored as JSON and read from request.body. Let me know if it still doesn't make sense.

Reply:
I think we will read it from the request body. Thanks! Will make sure to fix here #89

Reply:
Can we add a check for empty comments here? It seems that the user is able to post blank comments at the moment.

Reply:
Yes, will do!

Reply:
383282d3d3455e7d5fbee38772595495180fe1a5
Done.